### DISCUSSION TITLE
x-trap.noscroll not working after modal close

### DISCUSSION THREAD

**Discussion by cihantas** [alpinejs/alpine#3435] [7. Bugs (archived)]
I'm using Laravel Jetstreams modal which relies on x-trap.inert.noscroll to avoid scrolling when the modal is open. 

x-trap add `style="overflow: hidden; padding-right: 0px;"` to the `<html>` tag when the modal is first opened and leaves `style=""` when the modal is closed. But scrolling is still not possible. 

If you then manually add `overflow: hidden;` to the tag again and disable it, it works.

**Comment by ekwoka** [comment]
To me this seems like a browser bug.

Can you provide a demo?

**Comment by sanderjn** [comment]
I've come across this problem as well.

**Comment by AlbertRF147** [comment]
Same issue here. I think `"x-trap.noscroll": "false"` should remove the `overflow: hidden` css prop on the `html` element, but for some reason it doesn't.

**Comment by yungifez** [comment]
Possibly related to https://github.com/alpinejs/alpine/discussions/4317
